src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,44 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+export const TodoModal: React.FC<{
+  initialTodo: Todo
+  setCarierTodo: React.Dispatch<React.SetStateAction<Todo>>
+  carierTodo: Todo
+}> = ({
+  initialTodo,
+  setCarierTodo,
+  carierTodo,
+}) => {
+  const [user, setUser] = useState<User>();
+
+  const {
+    title, userId, id, completed,
+  } = carierTodo;
+
+  useEffect(() => {
+    try {
+      getUser(userId)
+        .then((userFromServer) => {
+          setUser(userFromServer);
+        });
+    } catch (error) {
+      <Loader />;
```

Comment:
the components should be here. please remove it
```suggestion
```

Reply:
I remove it on 29 line (TodoModal.tsx) and 30 line on TodoInfo.tsx. But I cant remove hook loader for TodoModal.tsx, becouse I use Loader on 101 line. User must see Loader when data loading.